test/index.test.ts

```diff
@@ -101,4 +98,41 @@ describe('Recover the address function of a transaction', () => {
       expect(recoveredAddress).not.toBe(signingAddress);
     });
   });
+
+  it('should recover the adress even if the message is an object', () => {
+    const lsp6Signer = new LSP6Signer();
+    testCases.forEach((data) => {
+      const messageData = lsp6Signer.sign(data, signingKey);
+      const signature = messageData.signature;
+      const isMessagePrefixed = true;
+      const recoveredAddress = lsp6Signer.recover(
+        messageData,
+        signature,
+        isMessagePrefixed,
+      );
+      expect(recoveredAddress).toBe(signingAddress);
+    });
+  });
+
+  it('should be prefixed with "\x19LSP6 Execute Relay Call:\n"', () => {
+    const lsp6Signer = new LSP6Signer();
+
+    const rawData = 'hello';
```

Comment:
You can use the same testCases.forEach pattern here so we cover more cases

Reply:
Like here:
https://github.com/ERC725Alliance/erc725.js/blob/develop/src/index.test.ts#L385
https://github.com/ERC725Alliance/erc725.js/blob/develop/src/index.test.ts#L1039